### ISSUE TITLE
Feature request: electron-server

### ISSUE THREAD

**Issue by derekchiang** [electron/electron#11506]
Let's say I've been using [lightgallery](https://electronjs.org/apps/lightgallery) for viewing my photos on my laptop.  One day I decide to switch to using a Chromebook.  Now, where do my photos go?   I decide to store them on an instance on AWS.  But now I've lost the ability to use `lightgallery` for my photos.

But what if there was an `electron-server` command that's exactly the same as `electron`, except that instead of launching a window, it launches a web server that serves the app over the network?

That is, I would simply `ssh` into my AWS instance, clone the `lightgallery` repo, and then run `electron-server main.js`.  Now I can go to `http://server-ip:8000`, and lightgallery is right there with my photos!

In other words, `electron-server` makes it possible to serve unmodified Electron app over the network.

**Comment by derekchiang** [comment]
What I gave was just one example use case.  There are many more reasons why you might want to run a web server which stores your data, and run electron apps over the network:

* To share an electron app (and the associated data) among a group of friends.
* To access an electron app (and the associated data) remotely and from multiple devices, e.g. a public computer.

**Comment by MarshallOfSound** [comment]
This kind of thing could be implemented yourself but I would warn against it.  Giving a remote location privileged and unrestricted access to your machine is a disaster waiting to happen.  I can say that this feature will almost certainly not be implemented by us.
